src/main/java/xyz/vegaone/easytracking/mapper/TagMapper.java

```diff
@@ -0,0 +1,15 @@
+package xyz.vegaone.easytracking.mapper;
+
+import org.mapstruct.Mapper;
+import xyz.vegaone.easytracking.domain.TagEntity;
+import xyz.vegaone.easytracking.dto.Tag;
+
+import java.util.List;
+
+@Mapper (componentModel = "Spring")
+public interface TagMapper {
+    Tag domainToDto (TagEntity tagEntity);
```

Comment:
you have a space between the method name and the parameter parenthesis. Pls use CTRL+ALT+L to automatically format de code and CTRL+ALT+O to remove any unused imports